tests/tests.js

```diff
@@ -4,42 +4,125 @@ const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee'],
-      coverageLevel: [{ role: 'ee', coverage: 125000 }],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
-
-    expect(price).to.equal(39.37)
+describe('pricing', () => {
+  describe('formatPrice', () => {
+    it('return the price with 2 decimal places', () => {})
+    let formatprice = pricing.formatPrice(33.323443)
+    expect(formatprice).to.equal(33.32)
   })
 
-  it('returns the price for a voluntary life product for an employee with a spouse', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee', 'sp'],
-      coverageLevel: [
-        { role: 'ee', coverage: 200000 },
-        { role: 'sp', coverage: 75000 },
-      ],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
-
-    expect(price).to.equal(71.09)
+  describe('getEmployerContribution', () => {
+    it('return contribution if employee has contributed ', () => {
+      const contributionMode = products.longTermDisability.employerContribution
+      const price = 43.75
+      const contribution = pricing.getEmployerContribution(
+        contributionMode,
+        price
+      )
+      expect(contribution).to.equal(10)
+    })
+    it('return contribution if employee has not contributed', () => {
+      const contributionMode = products.voluntaryLife.employerContribution
+      const price = 43.75
+      const contribution = pricing.getEmployerContribution(
+        contributionMode,
+        price
+      )
+      expect(contribution).to.equal(4.375)
+    })
   })
```

Comment:
Employer contribution is about whether the contribution is a dollar amount or a percentage of the price. Your tests should talk about that since that's what the function does. Always read the code to know what it's doing.